File: Backend/Domain/JEdwards.Domain/Api/MovieFullInfo.cs
namespace JEdwards.Domain.Api
{
    public record MovieFullInfo(string Title, string Year, string ImdbID, string Type, string Poster, string Rated, string Released, string Runtime, string Genre, string Director, string Writer, string Actors, string Plot, string Language, string Country, string Awards, List<Rating> Ratings, int Metascore, float ImdbRating, string ImdbVotes, string DVD, string BoxOffice, string Production, string Website, string Response) : Movie(Title, Year, ImdbID, Type, Poster);
}

File: jedwards.client/src/app/models/MovieFullInfo.ts
import { Movie } from "./Movie";
import { Rating } from "./Rating";

export class MovieFullInfo extends Movie {
    rated!: string;
    released!: string;
    runtime!: string;
    genre!: string;
    director!: string;
    writer!: string;
    actors!: string;
    plot!: string;
    language!: string;
    country!: string;
    awards!: string;
    ratings: Rating[] | undefined
    metascore!: number;
    imdbRating!: number;
    imdbVotes!: string;
    dvd!: string;
    boxOffice!: string;
    production!: string;
    website!: string;
    response!: string;
}